Code/skyrim_ui/src/app/components/settings/settings.component.scss

```diff
@@ -9,7 +9,6 @@
 
 .container {
   overflow: auto;
-  max-height: 60vh;
```

Comment:
Why did you make this change?

Reply:
When the font size increased you would need to use the the scrollbar, which currently do not work. This makes sure you can actually reach the font size option.

Reply:
But without the max-height the overflow will not trigger and when the container gets too big (or the screen too small) it just gets cut at the bottom

Reply:
Yes, I see your point now. I'll add it back with enough room for the font options.

Reply:
Scrolling with the scrollwheel works fine so it should be reachable anyways. But putting it above the Debug section would be a good idea

Reply:
I think moving it ot the top would be best, since it's probably the one of the first things you want to change.